--- a/chrome/common/chrome_features.cc
+++ b/chrome/common/chrome_features.cc
@@ -1003,7 +1003,7 @@ BASE_FEATURE(kHttpsFirstModeIncognitoNew
 #if BUILDFLAG(IS_MAC)
 // Enables immersive fullscreen. The tab strip and toolbar are placed underneath
 // the titlebar. The tab strip and toolbar can auto hide and reveal.
-BASE_FEATURE(kImmersiveFullscreen, base::FEATURE_ENABLED_BY_DEFAULT);
+BASE_FEATURE(kImmersiveFullscreen, base::FEATURE_DISABLED_BY_DEFAULT);
 
 // Enables immersive fullscreen mode for PWA windows. PWA windows will use
 // immersive fullscreen mode if and only if both this and kImmersiveFullscreen
