--- a/chrome/common/chrome_features.cc
+++ b/chrome/common/chrome_features.cc
@@ -897,7 +897,7 @@ BASE_FEATURE(kHttpsFirstModeIncognitoNew
 // the titlebar. The tab strip and toolbar can auto hide and reveal.
 BASE_FEATURE(kImmersiveFullscreen,
              "ImmersiveFullscreen",
-             base::FEATURE_ENABLED_BY_DEFAULT);
+             base::FEATURE_DISABLED_BY_DEFAULT);
 
 // Enables immersive fullscreen mode for PWA windows. PWA windows will use
 // immersive fullscreen mode if and only if both this and kImmersiveFullscreen
